docs/installation/windows.md

```diff
@@ -31,16 +31,17 @@ small ~24MB download, and boots in approximately 5s.
 
 ## Requirements
 
-Your machine must be running Windows 7.1, 8/8.1 or newer to run Docker. Windows 10 is not currently supported. To find out what version of Windows you have:
+Your machine must be running Windows 7, 8/8.1 or newer to run Docker. Windows 10 is not currently supported. To find out what version of Windows you have:
```

Comment:
Oh, we fixed that 7.1 before in another PR, must have gone lost somehow in a rebase. Thanks!